benchmarks/setup/default/worker.yaml

```diff
@@ -31,11 +31,11 @@ spec:
             value: "warn"
         resources:
           limits:
-            cpu: 4
-            memory: 2Gi
+            cpu: 100m
```

Comment:
:question: Same - 50m to 100m seem super low, can we actually keep up with the rate?

Reply:
Did you checked the benchmark ? :) For me it looked like it :) What would be your suggestion?

Reply:
So with less than 1 CPU, it means we have effectively no parallelism right? You can only process one thing at a time. That includes garbage collection (though I suspect with 100m the serial GC is used?). Does this mean during garbage collection, the whole gateway stops? I guess it's probably negligible?

I would err on the side of caution and have a higher limit if only to handle bursts when we need it. wdyt?

Reply:
> So with less than 1 CPU, it means we have effectively no parallelism right? You can only process one thing at a 
time. 

No this is not really true. The CPU resource is divided in shares and if you have for example 100m, means the app is allowed to use 100 CPU miliseconds in one second. You still can have multiple threads, this would just cause to reach the quota faster. Interesting read about it here https://learnk8s.io/setting-cpu-memory-limits-requests

I think we can go to 1 or 500m and this should be fine, we saw no issue in the benchmark before. 

We could also try out https://github.com/kubernetes/autoscaler/tree/master/vertical-pod-autoscaler to find the right values :)

Reply:
> That includes garbage collection (though I suspect with 100m the serial GC is used?). Does this mean during garbage collection, the whole gateway stops? I guess it's probably negligible?

I think you mean the worker right? Yes this is true